packages/@webex/plugin-cc/src/AgentConfigService/types.ts

```diff
@@ -0,0 +1,232 @@
+/**
+ * Represents the request to UserRequest.
+ *
+ * @public
+ */
+
+export interface UserRequest {
```

Comment:
Where are we using htis?

Reply:
We can remove this interface. Code Updated!